packages/pgi/package.py

```diff
@@ -0,0 +1,17 @@
+from spack import *
+from spack.util.prefix import Prefix
+import os
+
+class Pgi(Package):
```

Comment:
Put inheritance here at least, otherwise it will be really bug proned

Reply:
Undesirable, as built-in PGI package is currently unusable (needs a license and manual install).